src/main/java/io/cdap/e2e/pages/locators/CdfPluginPropertiesLocators.java

```diff
@@ -243,4 +243,12 @@ public static By reviewAssessment() {
   public static By configureProperties() {
     return By.xpath("//h3[contains(text(), 'Configure advanced properties')]");
   }
+
+  public static WebElement selectReplicationTable(String tableName) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//div[contains(text(),'" + tableName + "')]" +
```

Comment:
Change the code as given below, it improves the readability. 
```suggestion
    String path = "//div[contains(text(),'" + tableName + "')]" + "/preceding-sibling::div/span";
    return SeleniumDriver
    .getDriver()
    .findElement(By.xpath(path));
```

Reply:
Done.